liquibase-integration-tests/src/test/resources/changelogs/common/common.tests.changelog.xml

```diff
@@ -1177,4 +1177,22 @@
         <dropView viewName="does_not_exist" ifExists="true"/>
     </changeSet>
 
+    <changeSet id="relative import" author="mallod">
+        <createTable tableName="my_table">
+            <column name="test_col" type="int"/>
+            <column name="other_col" type="varchar(100)"/>
+        </createTable>
+        <loadData tableName="my_table"
+                  file="computedValueData.csv"
+                  relativeToChangelogFile="true">
+            <column name="test_col" type="COMPUTED"/>
+        </loadData>
```

Comment:
@filipelautert , do you think it is worthwhile to add a test for `loadUpdateData`? 

[https://docs.liquibase.com/change-types/load-update-data.html](https://docs.liquibase.com/change-types/load-update-data.html)

Reply:
@XDelphiGrl I think it would be pretty much the same than the changeset added for loadData, but it's not a big deal adding an extra changeset for testing loadUpdateData. So, I'm adding it now.